### ISSUE TITLE
[Bug]: Animal Openpose does not work on Google Colab

### ISSUE THREAD

**Issue by sst030164** [Mikubill/sd-webui-controlnet#2697]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

Animal Openpose does not reflect to the generated pictures.

![image](https://github.com/Mikubill/sd-webui-controlnet/assets/158999513/8fe5a772-09cd-4c18-8bce-b60e7b8c442c)

### Steps to reproduce the problem

1. Create new notebook of Google Colab
2. !git clone https://github.com/AUTOMATIC1111/stable-diffusion-webui
3. %cd /content/stable-diffusion-webui
  !python launch.py --share --enable-insecure-extension-access
4. Install extension from the following URL and [Apply and Quit] (WebUI will stop)
  https://github.com/Mikubill/sd-webui-controlnet
5. !curl -Lo /content/stable-diffusion-webui/extensions/sd-webui-controlnet/models/control_sd15_animal_openpose_fp16.pth https://huggingface.co/huchenlei/animal_openpose/blob/main/control_sd15_animal_openpose_fp16.pth
6. Start WebUI again
7. Click ControlNet and Choose ControlNet Unit 1 or 2
8. Put source image in canvas of ControlNet
9. Check [Enable] in ControlNet
10. Choose Preprocessor [animal_openpose] and Model [control_sd15_animal_openpose_fp16]
11. Run Processor and "Preprocessor Preview(Key line)" generate
12. Input prompt (like as "cat") and [Generate]

### What should have happened?

The openpose reflected picture is generated.

### Commit where the problem happens

webui: v1.8.0
controlnet: v1.1.441

### Command Line Arguments

```Shell
Run Web UI by the following command
!python launch.py --share --enable-insecure-extension-access
```

### List of enabled extensions

![image](https://github.com/Mikubill/sd-webui-controlnet/assets/158999513/74f8a720-4dec-460b-ac0d-7bd9b85cdfe8)

### Console logs

```Shell
/content/stable-diffusion-webui
Python 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]
Version: v1.8.0
Commit hash: bef51aed032c0aaa5cfd80445bc4cf0d85b408b5
ControlNet init warning: Unable to install insightface automatically. Please try run `pip install insightface` manually.
Launching Web UI with arguments: --share --enable-insecure-extension-access
2024-03-20 03:15:21.489941: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:9261] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2024-03-20 03:15:21.489993: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:607] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2024-03-20 03:15:21.491401: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1515] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2024-03-20 03:15:22.808808: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
no module 'xformers'. Processing without...
no module 'xformers'. Processing without...
No module 'xformers'. Proceeding without it.
ControlNet preprocessor location: /content/stable-diffusion-webui/extensions/sd-webui-controlnet/annotator/downloads
2024-03-20 03:15:27,371 - ControlNet - INFO - ControlNet v1.1.441
2024-03-20 03:15:27,587 - ControlNet - INFO - ControlNet v1.1.441
Loading weights [6ce0161689] from /content/stable-diffusion-webui/models/Stable-diffusion/v1-5-pruned-emaonly.safetensors
2024-03-20 03:15:27,968 - ControlNet - INFO - ControlNet UI callback registered.
Running on local URL:  http://127.0.0.1:7860
Creating model from config: /content/stable-diffusion-webui/configs/v1-inference.yaml
Applying attention optimization: Doggettx... done.
Model loaded in 6.7s (load weights from disk: 1.2s, create model: 0.5s, apply weights to model: 4.2s, apply half(): 0.1s, load textual inversion embeddings: 0.4s, calculate empty prompt: 0.2s).
Running on public URL: https://fd8ff98ae30d88c773.gradio.live

This share link expires in 72 hours. For free permanent hosting and GPU upgrades, run `gradio deploy` from Terminal to deploy to Spaces (https://huggingface.co/spaces)
Startup time: 26.7s (prepare environment: 3.8s, import torch: 5.7s, import gradio: 1.2s, setup paths: 4.0s, initialize shared: 0.2s, other imports: 1.1s, load scripts: 1.5s, create ui: 0.7s, gradio launch: 8.5s).
2024-03-20 03:19:51,968 - ControlNet - INFO - Preview Resolution = 512
Downloading: "https://huggingface.co/yzd-v/DWPose/resolve/main/yolox_l.onnx" to /content/stable-diffusion-webui/extensions/sd-webui-controlnet/annotator/downloads/openpose/yolox_l.onnx

100% 207M/207M [00:00<00:00, 308MB/s]
Downloading: "https://huggingface.co/bdsqlsz/qinglong_controlnet-lllite/resolve/main/Annotators/rtmpose-m_simcc-ap10k_pt-aic-coco_210e-256x256-7a041aa1_20230206.onnx" to /content/stable-diffusion-webui/extensions/sd-webui-controlnet/annotator/downloads/openpose/rtmpose-m_simcc-ap10k_pt-aic-coco_210e-256x256-7a041aa1_20230206.onnx

100% 52.0M/52.0M [00:00<00:00, 73.8MB/s]
2024-03-20 03:21:20,848 - ControlNet - INFO - unit_separate = False, style_align = False
2024-03-20 03:21:21,225 - ControlNet - INFO - Loading model: control_sd15_animal_openpose_fp16 [e3b0c442]
*** Error running process: /content/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/controlnet.py
    Traceback (most recent call last):
      File "/content/stable-diffusion-webui/modules/scripts.py", line 784, in process
        script.process(p, *script_args)
      File "/content/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/controlnet.py", line 1279, in process
        self.controlnet_hack(p)
      File "/content/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/controlnet.py", line 1264, in controlnet_hack
        self.controlnet_main_entry(p)
      File "/content/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/controlnet.py", line 936, in controlnet_main_entry
        model_net, control_model_type = Script.load_control_model(p, unet, unit.model)
      File "/content/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/controlnet.py", line 367, in load_control_model
        control_model = Script.build_control_model(p, unet, model)
      File "/content/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/controlnet.py", line 395, in build_control_model
        state_dict = load_state_dict(model_path)
      File "/content/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/utils.py", line 22, in load_state_dict
        state_dict = unsafe_torch_load(ckpt_path, map_location=torch.device(location))
      File "/usr/local/lib/python3.10/dist-packages/torch/serialization.py", line 1040, in load
        return _legacy_load(opened_file, map_location, pickle_module, **pickle_load_args)
      File "/usr/local/lib/python3.10/dist-packages/torch/serialization.py", line 1258, in _legacy_load
        magic_number = pickle_module.load(f, **pickle_load_args)
    _pickle.UnpicklingError: invalid load key, '<'.

---
  0% 0/20 [00:00<?, ?it/s]
  5% 1/20 [00:01<00:25,  1.34s/it]
 10% 2/20 [00:01<00:12,  1.47it/s]
 15% 3/20 [00:01<00:07,  2.16it/s]
 20% 4/20 [00:01<00:05,  2.79it/s]
 25% 5/20 [00:02<00:04,  3.34it/s]
 30% 6/20 [00:02<00:03,  3.78it/s]
 35% 7/20 [00:02<00:03,  4.03it/s]
 40% 8/20 [00:02<00:02,  4.30it/s]
 45% 9/20 [00:02<00:02,  4.51it/s]
 50% 10/20 [00:03<00:02,  4.66it/s]
 55% 11/20 [00:03<00:01,  4.77it/s]
 60% 12/20 [00:03<00:01,  4.82it/s]
 65% 13/20 [00:03<00:01,  4.80it/s]
 70% 14/20 [00:03<00:01,  4.83it/s]
 75% 15/20 [00:04<00:01,  4.83it/s]
 80% 16/20 [00:04<00:00,  4.89it/s]
 85% 17/20 [00:04<00:00,  4.90it/s]
 90% 18/20 [00:04<00:00,  4.90it/s]
 95% 19/20 [00:04<00:00,  4.88it/s]
100% 20/20 [00:05<00:00,  3.85it/s]

Total progress: 100% 20/20 [00:04<00:00,  4.76it/s]
```

**Comment by sst030164** [comment]
I update the subject because this issue happed not only animal_openpose but also the other ControlNet such as Openpose, Depth, etc.

**Comment by dilectiogames** [comment]
for the past 12 months google had actively messed up the backed python extensions to "soft ban" & "shadow ban" AI image generation. this way it appears is your fault or there is some bug you need to figure out. corporate gaslight at it finest.

the way I was "fixing" this "issue" was to reinstall the whole thing all over again. I've got good and fast at this at some point, had a modified script that set everything up automatically, I've moved the heavy models out of the colab before deleting the notebook then added them back to avoid waiting for huge downloads al the time.

Everytime this mysteriously "fixed" the issue. I didn't generated a lot of images or anything, and I've not used their service to generate nsfw of any kind. So I was not spamming their service. But that didn't matter for google. At first everything broke after a week then a few days then the same day after a few hours. in the end they have won, it was too much work for me to deal with this bs.

Long story short. try my "solution". let us know the results

**Comment by sdbds** [comment]
It's a Google colab issue, Google has been banning free users of stable diffusion for a long time and doesn't know what changes they made to the python environment to disable free users.
